models/ACC/ACC.jl

```diff
@@ -0,0 +1,29 @@
+using ReachabilityAnalysis
+
+@taylorize function ACC!(dx, x, p, t)
+    local u, a_lead, a_ego = 0.0001, -2., -1.
```

Comment:
where did you take the value `a_ego`? i can't find it in the problem's description. actually seems to me that it should be zero, because it says

> For this case
study, the ego car is set to travel at a set speed V set = 30 and maintains a safe distance D saf e
from the lead car.

as it the ego car travels at constant speed.

Reply:
`a_ego` is the output of the NN, i put the `-1` to have a placeholder

Reply:
ok i see, you're right, i take my comment back